### PR TITLE
issues/374

### PR DESCRIPTION
**Pull request by goern**
- add a stage deployment on nostromo
- add mosquitto to stage deployment
- move the mosquitto sealed secret to the env, as sealed secrets can only be decrypted on the env they have been encrypted for
- add a sealed secret for mosquitto stage deployment on nostromo
- re-encrypted mosquitto secret
- renamed secret files, add influxdb2-auth secret for nostromo stage deployment, ⚠️ secrets/ directory is no longer ignored by git
- ☠ remove smaug environment, that cluster is dead!
- ✨ bounce telegraf version
- ✨ fix some E721 errors, https://www.flake8rules.com/rules/E721.html
- update pre-commit to be used
- Bump tornado from 6.3.2 to 6.3.3 in /notebooks (#371)
- Bump tornado from 6.3.2 to 6.3.3 in /components/paddock (#372)
- add car class (#379)
- ✨ update pipelines, update telegraf config, see #374

### CODE REVIEW

File: manifests/env/nostromo-stage/telegraf/telegraf.conf [hunk 1]
```diff
@@ -102,13 +86,12 @@ tags = ["CurrentLap", "CarClass"]
...
 token = "$INFLUX_TOKEN"
 urls = ["http://influxdb2:8086"]
-namepass = ["laps*"]
 
 # Configuration for the Prometheus client to spawn
 [[outputs.prometheus_client]]
-## Address to listen on.
 listen = ":9090"
-namepass = ["internal_*"]
+namepass = ["*"]
```
Comment (durandom) [inline manifests/env/nostromo-stage/telegraf/telegraf.conf:97]: let's keep this. IIRC without this, we also expose the racing telemetry to prometheus

File: manifests/env/nostromo-stage/telegraf/telegraf.conf [hunk 2]
```diff
@@ -102,13 +86,12 @@ tags = ["CurrentLap", "CarClass"]
 
 [[outputs.influxdb_v2]]
 bucket = "racing"
+namepass = ["laps*"]
 organization = "b4mad"
 token = "$INFLUX_TOKEN"
 urls = ["http://influxdb2:8086"]
-namepass = ["laps*"]
```
Comment (durandom) [inline manifests/env/nostromo-stage/telegraf/telegraf.conf:155]: and without this, we put the metrics meant for prometheus into influx

File: manifests/base/influxdb/influx-config.yaml [hunk 3]
```diff
@@ -2,7 +2,6 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: influx-config
-  namespace: b4mad-racing-test
```
Comment (durandom) [inline manifests/base/influxdb/influx-config.yaml:5]: that will be added again by running
```
helm template influxdb2 bitnami/influxdb \
  --values influxdb2-values.yaml \
  --output-dir influxdb2
```

in the future. 🤷

File: manifests/env/phobos/kustomization.yaml [hunk 4]
```diff
@@ -70,3 +71,5 @@ images:
   - name: paddock
     newName: image-registry.openshift-image-registry.svc:5000/b4mad-racing/paddock
     newTag: latest
+  - name: telegraf
+    newTag: "1.27.3"
```
Comment (durandom) [inline manifests/env/phobos/kustomization.yaml:75]: shouldnt we keep it at the previous version until we know that it works on nostromo?

**Review by durandom — COMMENTED** [review]
theres a lot going on here :)

looks good to me

**Comment by goern** [comment]
@durandom this is a mega-patch, please have a look at it. It basically deploys a stage environment (w/o grafana)
only known issue is the github login flow, as the redirect url is not pointing to the stage URL
some minor cleanup in the base/ directory
/unhold

File: manifests/base/influxdb/influx-config.yaml (hunk 3 above)
Comment (goern) [reply manifests/base/influxdb/influx-config.yaml:5]: ja, so re need to remove it again in the future :P

File: manifests/env/phobos/kustomization.yaml (hunk 4 above)
Comment (goern) [reply manifests/env/phobos/kustomization.yaml:75]: exactly why I started deploying nostromo-stage

File: manifests/env/nostromo-stage/telegraf/telegraf.conf (hunk 2 above)
Comment (goern) [reply manifests/env/nostromo-stage/telegraf/telegraf.conf:155]: pls recheck e78a167

**Review by durandom — APPROVED** [review]
LGTM

/approve

**Pull request merged**